packages/react/package.json

```diff
@@ -24,8 +24,8 @@
     "dist/"
   ],
   "dependencies": {
-    "@ukic/fonts": "^2.3.0",
-    "@ukic/web-components": "^2.4.0"
```

Comment:
This doesn't seem to be covered by the ticket changes, is this intentional?

Reply:
The dependencies are locked down to prevent the web-components and react packages not being in tandem. For example, there are issues if the react package is 2.4.0 and the downloaded web-components package is 2.2.0 for example. This is to prevent that  from happening.